app/aboutdialog.cpp

```diff
@@ -44,7 +44,7 @@ AboutDialog::AboutDialog(QWidget *parent)
         (QStringLiteral("<br/>") + tr("Version: %1").arg(GIT_DESCRIBE_VERSION_STRING)),
 #endif // GIT_DESCRIBE_VERSION_STRING
         QStringLiteral("<hr/>"),
-        tr("Copyright (c) 2020 %1").arg(QStringLiteral("<a href='https://github.com/BLumia'>@BLumia</a>")),
+        tr("Copyright © 2020 %1").arg(QStringLiteral("<a href='https://github.com/BLumia'>@BLumia</a>")),

```

Comment:
`©` is also not a part of the ASCII character set, so in the original text for `LANG=C`, we prefer to use `(c)`.